connexion/security/async_security_handler_factory.py

```diff
@@ -62,12 +62,12 @@ async def wrapper(request, token, required_scopes):
         return wrapper
 
     @classmethod
-    def verify_security(cls, auth_funcs, required_scopes, function):
+    def verify_security(cls, auth_funcs, function):
```

Comment:
Shouldn't we apply the same updates to this function? It overwrites the one in the `AbstractSecurityHandlerFactory`.